<ui version="4.0" >
 <class>streamForm</class>
 <widget class="QWidget" name="streamForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Video Stream</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="playB" >
         <property name="minimumSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Play/Pause</string>
         </property>
         <property name="whatsThis" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="captureB" >
         <property name="minimumSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Capture Image</string>
         </property>
         <property name="whatsThis" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="imgFormatCombo" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Image Format</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>116</width>
           <height>57</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="videoFrame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
